examples/seismic/acoustic/acoustic_example.py

```diff
@@ -53,24 +50,25 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
     # checkpointing, PyRevolve will take care of the time history
     save = full_run and not checkpointing
     # Define receiver geometry (spread across x, just below surface)
-    rec, u, summary = solver.forward(save=save, autotune=autotune)
+    rec, u, summary = solver.forward(save=save, autotune=autotune, kernel=kernel)
```

Comment:
I still think this is more than odd, but it's entirely possible I'm just wrong. So I'd like someone's input as well. For example, @rhodrin , what do you think? should ``kernel`` be a .forward/.backward argument, or a solver (``acoustic_setup(...)`` ) argument?

Maybe it's the name "acoustic_setup" that confuses me?